third_party/blink/renderer/core/events/event_type_names.json5

```diff
@@ -45,13 +45,15 @@
     "backgroundfetchfail",
     "backgroundfetchsuccess",
     "beforecopy",
+    "beforeconceal",
     "beforecreatepolicy",
     "beforecut",
     "beforeinput",
     "beforeinstallprompt",
     "beforematch",
     "beforepaste",
     "beforeprint",
+    "beforestop",
```

Comment:
@andrewsavage1 has a plan to extract Cobalt-specific events out of this upstream file (as seen in #4993) - can you add a ` TODO(b/400518318)` in these two plz?